docs/v0.2.0/design/gatewayapi-support.md

```diff
@@ -0,0 +1,76 @@
+# Gateway API Support
+
+As mentioned in the [SystemDesign][] document, Envoy Gateway's managed data plane is configured dynamically through 
+Kubernetes resources, primarily [Gateway API][] objects. Envoy Gateway supports configuration using the following Gateway API resources.
+
+## **GatewayClass**
+
+[GatewayClasses][] are used to configure which Gateways and other reliant resources should be managed by Envoy Gateway.
```

Comment:
can we add a comment mentioning that EG supports/accepts a single GatewayClass resource linked to the EG controller, and accepts in order of age (oldest first)

Reply:
s/[GatewayClasses][] are/A [GatewayClass][] is/